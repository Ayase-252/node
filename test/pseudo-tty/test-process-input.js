'use strict'

const common = require('../common');
const assert = require('assert');

process.input('question here').then(common.mustCall((res) => {
  assert.strictEqual(res, 'answer');
}));
