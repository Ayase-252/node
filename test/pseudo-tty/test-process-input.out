answer
[1G[0Jquestion here[14Ganswer
